### PR TITLE
minor update to publication

### LINKED ISSUE #20: refine publication part
currently publication is too much involved with `github action` and possibly not easy to test out in the session.

suggect to use the ordinary `pkgdown::build_site` and `pkgdown::deploy` instead

### PR DESCRIPTION
**Pull request by clarkliming**
closes #20 

add `pkgdown` here for manual deployment

### CODE REVIEW

File: slides/06_publication.qmd [hunk 1]
```diff
@@ -222,8 +223,11 @@ Maintainer: 'Friedrich Pahlke'
 
 # Exercise
 
+Use the same repository created from template https://github.com/kkmann/simulatr
```
Comment (danielinteractive) [inline slides/06_publication.qmd]:
```suggestion
Use the same repository created from template [https://github.com/kkmann/simulatr](https://github.com/kkmann/simulatr)
```

File: slides/06_publication.qmd [hunk 2]
```diff
@@ -222,8 +223,11 @@ Maintainer: 'Friedrich Pahlke'
...
+
 - Run CRAN like checks in RStudio: use `--as-cran` in check options
 - Look at the CI/CD checks reported on your GitHub repository
   - Where are the `R CMD check` results?
-- Deploy the `pkgdown` website with `usethis`
-
+- Deploy the website with `pkgdown`
```
Comment (danielinteractive) [inline slides/06_publication.qmd:232]: can we still reference `usethis` as alternative? that can be preferred by some participants/teachers

Comment (clarkliming) [reply slides/06_publication.qmd:232]: here if we are not involving much with the github actions, then we have to use `pkgdown::deploy_to_branch` functionality.
`usethis::use_pkgdown` set up the _pkgdown.yaml file, `usethis::use_pkgdown_github` set up github repository to use `gh-pages` be the site etc, `usethis::use_github_action` to enable the github action to deploy the website. If we are going to set up the github actions then we can still use `usethis` to do that

File: slides/06_publication.qmd [hunk 3]
```diff
@@ -222,8 +223,11 @@ Maintainer: 'Friedrich Pahlke'
...
+Use the same repository created from template [https://github.com/kkmann/simulatr](https://github.com/kkmann/simulatr)
+
 - Run CRAN like checks in RStudio: use `--as-cran` in check options
 - Look at the CI/CD checks reported on your GitHub repository
   - Where are the `R CMD check` results?
-- Deploy the `pkgdown` website with `usethis`
-
+- Deploy the website with `pkgdown`
+  - Need to change remote to `git`
+  - (Optional) Check the automatic github page deploy action
```
Comment (clarkliming) [inline slides/06_publication.qmd:235]: here I put the github action as optional

File: slides/06_publication.qmd (hunk 2 above)
Comment (danielinteractive) [reply slides/06_publication.qmd:232]: Yeah I think here we can mention both options, via pkgdown and via usethis

**Review by danielinteractive — APPROVED** [review]

**Pull request merged**